packages/wdio-spec-reporter/tests/index.test.ts

```diff
@@ -395,7 +396,7 @@ describe('SpecReporter', () => {
             expect(result).toMatchSnapshot()
         })
 
-        it('should not print if data table format is not given', () => {
+        it('should not print if argument is a single line doc string', () => {
```

Comment:
I've changed the name and the expected result of the `should not print if data table format is not given` test. The existing name and expectation are not appropriate when supporting doc strings which is a string. It will be covered by the new test case `should not print if the argument is undefined`.